.github/workflows/release.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
       - name: Run chart-release
-        uses: helm/chart-releaser-action@cae68fefc6b5f367a0275617c9f83181ba54714f # v1.7.0
+        uses: helm/chart-releaser-action@v1.7.0
```

Comment:
Helm Chart Releaser does not have a verified creator badge.
I'm not sure whether it ever had a badge or never had one.
https://github.com/marketplace?query=Helm+Chart+Releaser&verification=verified_creator&type=actions

v1.7.0 is not investigated.
https://bozuman.cybozu.com/k/85298/show#record=15

Reply:
Thanks. I misunderstood about the verified creator.
I changed it back to hash and checked the v.17.0' s code.